### DISCUSSION TITLE
Adding notes to friends that would show on profile cards

### DISCUSSION THREAD

**Discussion by Skubers** [ppy/osu#25367] [Ideas]
Just a basic function to let users add notes to the friends that would appear on the friend's profile card, only for them.

It would be useful if you have many random friends, you're forgetful and don't remember who is who, or someone is constantly changing the nickname and you can't be bothered to check for that person's previous nickname

**Comment by Rekunan** [comment]
I'll add that Discord also has this feature:
![image](https://github.com/ppy/osu/assets/24501632/1aae2934-144b-439a-a92f-174da89cd81d)
